include/net/net_if.h

```diff
@@ -56,7 +56,8 @@ struct net_if_addr {
 
```

Comment:
In commit message "net/iface: Coalesce all DAD timers through one", the string "Each net_if_addr can be added to the timer handler via a slist, then." sounds a bit too French. What about just saying "Then each net_if_addr can be added to the timer handler via a slist."